### ISSUE TITLE
Scoreboard Request Feature

### ISSUE THREAD

**Issue by 96LawDawg** [ArnoldSmith86/virtualtabletop#1725]
It would be a significant re-write, but would it be possible to have the click-on scoreboard functionality be changed so that the cells act like labels?  Instead of popping up an overlay where you have to select multiple things, could you just select the cell in the row/column you want and manually enter a value?

**Comment by ArnoldSmith86** [comment]
Maybe a solution that doesn't need the keyboard would be nicer? Especially because it's pretty disruptive on mobile.

One idea would be to highlight the clicked cell, dimm everything else and show plus and minus buttons with different values (1,5,10,etc.) around it.

We could also change the current overlay to select the player and round of the clicked cell.

**Comment by 96LawDawg** [comment]
Good idea.  Or maybe instead of +/- 1, 5, 10, could a long press work?  It sort of works on PCIO (as long as you hold the mouse very still).

**Comment by ArnoldSmith86** [comment]
I think I would prefer showing a little number keypad. Should be less button presses overall, shouldn't it?

Hmm, but I guess it's worse when editing an existing entry..

**Comment by 96LawDawg** [comment]
We could have both a keypad and +/- buttons.
![Screenshot 2023-04-16 093725](https://user-images.githubusercontent.com/76912527/232320442-d56b9cfb-df53-4e6b-a13d-503664efc9b1.png)

**Comment by flackr** [comment]
Before you write a numeric keyboard, have you tried using the appropriate input hints to show a numeric-only keyboard on mobile? https://stackoverflow.com/questions/6178556/phone-numeric-keyboard-for-text-input

**Comment by ArnoldSmith86** [comment]
Problem is that it either resizes the room and everything shrinks to oblivion or the mobile keyboard overlaps the room, potentially covering the scoreboard.

**Comment by robartsd** [comment]
I like the idea of opening the input overlay with the selected cell.

**Comment by mousewax** [comment]
Keyboard pop ups usually don't resize the page and they appear over the content. I think a proper keyboard hint as suggested by flackr is the right thing to do. Barring that a popover appearing below or above the cell similar to what LawDawg posted is also preferable. The input overlays are very jarring. Scoresheets look and act like spreadsheets and they should probably act in a similar manner. When you look at them you already have an idea of how they probably work.

**Comment by mousewax** [comment]
And if we're doing pie in the sky things, many scorekeeper apps that I use on my phone and tablet allow you to do simple math within the input boxes. For example, in one round I can type "10+15+8-5" and it would record the proper value.
